spec/03-plugins/08-datadog/01-log_spec.lua

```diff
@@ -287,6 +314,41 @@ describe("Plugin: datadog (log)", function()
     assert.contains("kong.dd3.latency:%d+|g|#T2:V2:V3,T4", gauges, true)
   end)
 
+  it("logs metrics with tags with api name in tag", function()
```

Comment:
can we add a test to ensure that api_name tag is not added when `tag_api_name`  is false?

Reply:
Sure!